specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/keys.json

```diff
@@ -1473,7 +1473,7 @@
         },
         "exportable": {
           "type": "boolean",
-          "description": "Indicates if the private key can be exported."
+          "description": "Indicates if the private key can be exported. Release policy must be provided when creating 1st version of exportable key."
```

Comment:
Once you mark a key exportable = true, can you later set exportable = false?

Reply:
No, value for exportable flag can never be changed. Once, the 1st version of a key has exportable set to true, then all new versions of the key have to have same value for exportable.

Exportable is a global property that applies to all versions of a key.